scripts/zkai_attention_derived_d128_block_statement_chain_gate.py

```
@@ -0,0 +1,881 @@
+        raise AttentionDerivedD128BlockStatementChainError("summary drift")
+    if payload.get("non_claims") != NON_CLAIMS:
+        raise AttentionDerivedD128BlockStatementChainError("non-claims drift")
+    if payload.get("validation_commands") != VALIDATION_COMMANDS:
+        raise AttentionDerivedD128BlockStatementChainError("validation commands drift")
+    if payload.get("payload_commitment") != payload_commitment(payload):
+        raise AttentionDerivedD128BlockStatementChainError("payload commitment drift")
+    if keys == FINAL_KEYS:
+        inventory = _list(payload.get("mutation_inventory"), "mutation_inventory")
+        cases = _list(payload.get("cases"), "cases")
+        if tuple(inventory) != EXPECTED_MUTATIONS:
+            raise AttentionDerivedD128BlockStatementChainError("mutation inventory drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)